site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,106 @@
+
+Make sure you've created an Application in the link:/docs/v1/tech/core-concepts/applications[Application overview]. After, log in to your application by clicking on `Applications` in the navigation sidebar, selecting the green magnifying glass icon, and copying and pasting the `Login URL` into an incognito browser. Click the Forgot Password link, enter your email in the form, and you should see the `Forgot Password` email sent to your inbox.
+
+=== Overriding Email Templates using API
+
+Finally, we'll override our generic Tenant Email Template we just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the blue edit icon for the `Forgot Password` email template and copy the `HTML Template` `Text Template` sections into a notepad.
+
+Create a new email template by going to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
+Paste your default template from above into the `HTML Template`  and `Text Template` sections. Feel free to edit these templates based on your application.
+
+Save your template by clicking the blue save icon in the top right corner.
+
+=== Test the Custom Template
+
+For our demo application Pied Piper, we'll call the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Start Forgot Password Workflow] for an existing user. If you haven't already created a user, go ahead and complete the process in the  link:/docs/v1/tech/tutorials/register-user-login-api[Register a User and Login] guide.
+```shell
+curl --request POST \
+  --url https://localhost/api/user/forgot-password \
+  --header 'Authorization: xxxxxx' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "applicationId": "85a03867-dccf-4882-adde-1a79aeec50df",
+    "loginId": "dinesh@piedpiper.com",
+     "sendForgotPasswordEmail": true
+}'
+```
+
+Modify the curl command based on the values for your application and user.
+Once successfully executed, you should see your custom email sent to the above user's email address.
+
+=== Override Email Template using the Hosted Login Page
+
+In addition to calling the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Start Forgot Password Workflow] API, you can also append the query string parameter to your `Forgot Password` link on the FusionAuth link:/docs/v1/tech/core-concepts/integration-points#hosted-login-pages[hosted login screen].
+Grab your Login URL under [breadcrumb]#Applications -> click the green magnifying glass icon for your application -> copy a Login URL# and append `&applicationId=123`, replacing 123 with your [field]#applicationId#. Enter your email address and submit the form. You should again see your application email template for Forgot Password.
```

Comment:
```suggestion
Grab your Login URL under [breadcrumb]#Applications# by clicking the green magnifying glass icon for your application and copying the [uielement]#Login URL#. Append `&applicationId=YOUR_APPLICATION_ID`, replacing `YOUR_APPLICATION_ID` with your [field]#applicationId#. Enter your email address and submit the form. You should again see your Forgot Password application email content used .
```

Reply:
Is it `applicationId` or `clientId` or both?

Reply:
When you append the `applicationId` to the Login URL both would be included but the clientId is already included in the Login URL.